packages/@okta/i18n/src/properties/login.properties

```diff
@@ -601,6 +601,7 @@ primaryauth.newUser.tooltip.close=Close
 oktaVerify.description = To access this resource, your organization requires you to sign in with Okta FastPass.
 oktaVerify.appDescription = To access {0}, your organization requires you to sign in with Okta FastPass.
 oktaVerify.button = Sign in with Okta FastPass
+signinWithWebAuthn.button = Sign in with security key or biometric
```

Comment:
Just confirming, did you get the new properties looked at?

Reply:
@IanHakes-okta  can you please take a look at these two text?

Reply:
updated